boards/Board.mk

```diff
@@ -38,7 +38,7 @@ $(ETCSRC): $(addprefix $(BOARD_DIR)$(DELIM)src$(DELIM),$(RCRAWS)) $(RCOBJS)
 	$(Q) genromfs -f romfs.img -d $(ETCDIR)$(DELIM)$(CONFIG_NSH_ROMFSMOUNTPT) -V "NSHInitVol"
 	$(Q) echo "#include <nuttx/compiler.h>" > $@
 	$(Q) xxd -i romfs.img | sed -e "s/^unsigned char/const unsigned char aligned_data(4)/g" >> $@
-	$(Q) rm romfs.img
+	$(Q) rm -rf romfs.img $(ETCDIR)$(DELIM)
```

Comment:
the change will make romfs.img generate in each make even all content under etc/ isn't change.